app/views/books/_info_window.html.erb

```diff
@@ -0,0 +1,3 @@
+<h2><%= book.title %></h2>
```

Comment:
What is this new page about?
Who can see it?
Thanks

Reply:
it is part of the map. when you toggle a marker, you can see the Info window (in this case: The title of the book)

Reply:
Cool! thanks :)